signer-npm/js/src/index.js

```diff
@@ -93,7 +93,11 @@ function transactionSerializeRaw(message) {
 
   const gaspriceBigInt = new BN(message.gasprice, 10);
   const gaspriceBuffer = gaspriceBigInt.toArrayLike(Buffer, 'be', gaspriceBigInt.byteLength());
-  const gasprice = Buffer.concat([Buffer.from('00', 'hex'), gaspriceBuffer]);
+  let gasprice = Buffer.concat([Buffer.from('00', 'hex'), gaspriceBuffer]);
+
+  if (message.gasprice == "0") {
```

Comment:
is this `==` intended? should this be `===` instead?

Reply:
I am not sure if it is better `===` because it will work for `"0"` and `0` value when using `==` might have unintended behavior tho.